lib/matplotlib/backends/backend_qt5.py

```diff
@@ -1069,11 +1069,18 @@ def trigger(self, *args):
         QtWidgets.QMessageBox.information(None, "Help", self._get_help_html())
 
 
+class ToolCopyToClipboardQT(backend_tools.ToolCopyToClipboardBase):
+    def trigger(self, *args, **kwargs):
+        pixmap = self.canvas.grab()
+        qApp.clipboard().setPixmap(pixmap)
+
+
 backend_tools.ToolSaveFigure = SaveFigureQt
 backend_tools.ToolConfigureSubplots = ConfigureSubplotsQt
 backend_tools.ToolSetCursor = SetCursorQt
 backend_tools.ToolRubberband = RubberbandQt
 backend_tools.ToolHelp = HelpQt
+backend_tools.ToolCopyToClipboard = ToolCopyToClipboardQT
```

Comment:
All the backend-specific tool classes do not have the "Tool" prefix. While, in general I would recommend to keep the prefix also in the backend specific classes, in the present situation I'm +0.3 not using it for consistency with the existing code. But I'm not familiar with that part of the code and don't know if there are any explicit policies on this. Maybe someone more experienced with that code can comment on this?

Same holds for the other backends.

Reply:
@timhoffm you are right, we have to make the naming more consistent. 
We can make a separate PR fixing all those names at once. I don't think it is relevant here

Reply:
Do we consider the backend implementations private? If so we can clean up the naming soon and this PR goes in as is.

If not, we need a deprecation cycle. In that case, I‘d prefer to remove the Tool prefix in this PR as well and change everything later together.

Reply:
As the initial warning states. Tool names might change. So I don't see a problem doing all of them in a single PR